src/components/sbb-form-field-clear/sbb-form-field-clear.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  Component,
+  ComponentInterface,
+  Element,
+  Listen,
+  h,
+  Host,
+  JSX,
+  Prop,
+  State,
+} from '@stencil/core';
+import { ButtonProperties, resolveButtonRenderVariables } from '../../global/interfaces';
+import { hostContext } from '../../global/dom';
+import {
+  HandlerRepository,
+  actionElementHandlerAspect,
+  documentLanguage,
+  languageChangeHandlerAspect,
+} from '../../global/eventing';
+import { i18nClearInput } from '../../global/i18n';
+
+/**
+ * @slot unnamed - Slot to render the content.
+ */
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-form-field-clear.scss',
+  tag: 'sbb-form-field-clear',
+})
+export class SbbFormFieldClear implements ComponentInterface, ButtonProperties {
+  /**
+   * The name attribute to use for the clear button.
+   */
+  @Prop({ reflect: true }) public name: string | undefined;
```

Comment:
Does name property really make sense?

Reply:
I believe it is a required prop of the `ButtonProperties` interface.

Reply:
ok :) I just thought, in this case, it will never be used :D. maybe make it optional? But it's also ok as it is... @kyubisation  ?

Reply:
Technically it is always optional, even in other cases, because HTML does not require it.
Not sure whether we should change this here in isolation?

Reply:
So you would suggest to keep name here, or to change our own interface and make it optional?

Reply:
I just found this commit in the calendar pr: 
<img width="1352" alt="Bildschirmfoto 2023-09-05 um 09 01 02" src="https://github.com/lyne-design-system/lyne-components/assets/29278232/e27cce62-c1a2-47f1-951a-f5ae877182d6">

So I guess we should to the same here? @kyubisation

Reply:
you can remove name property and change interface as above.